PokeAlarm/Telegram/TelegramAlarm.py

```diff
@@ -105,7 +106,10 @@ def send_alert(self, alert, info, sticker_id=None):
     # Trigger an alert based on Pokemon info
     def pokemon_alert(self, pokemon_info):
         if self.__pokemon['stickers']:
-            self.send_alert(self.__pokemon, pokemon_info, sticker_list.get(str(pokemon_info['pkmn_id'])))
+            if str(pokemon_info['pkmn_id']) == '201':
+                self.send_alert(self.__pokemon, pokemon_info, unown_list.get(str(pokemon_info['form'])))
```

Comment:
I'd vote to include it into the sticker list file and rename it to ``sticker_list_unown`` to keep it clear what it is.